src/main/java/io/omnition/loadgenerator/model/topology/TagGenerator.java

```diff
@@ -0,0 +1,26 @@
+package io.omnition.loadgenerator.model.topology;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class TagGenerator {
+
+    private static final String defaultTagName = "tag-name-";
+    private static final String valName = "val-name-";
```

Comment:
this works and feel free to ignore my next line
I am wondering how docker create all those funny names for containers - if it is easy enough it might be cool

Reply:
iirc tables of adjectives x names it really gives a nice effect

Reply:
Did it for tag-name. For value, it was harder because we want to have variable length.